lib/StripeObject.php

```diff
@@ -13,45 +13,53 @@
 class StripeObject implements ArrayAccess, JsonSerializable
 {
     /**
-     * @var Util\Set Attributes that should not be sent to the API because
+     * @return Util\Set Attributes that should not be sent to the API because
      *    they're not updatable (e.g. API key, ID).
      */
-    public static $permanentAttributes;
+    public static function getPermanentAttributes()
+    {
+        static $permanentAttributes = null;
+        if ($permanentAttributes === null) {
+            $permanentAttributes = new Util\Set(array('_opts', 'id'));
+        }
```

Comment:
Oh man. I was going to say that it would be nice to just move this up to a statically initialized expression so we don't have to check the `null` everytime, but just reading through the manual it seems like this only became supported in 5.6 for non-trivial expressions :/

Reply:
Yep. I amended the commit so you can't see it, but my first submission used static initializations 😭